src/main/proto/inter_node_rpc_service.proto

```diff
@@ -92,9 +95,12 @@ enum MetricEnum {
   QUEUE_CAPACITY = 7 [(additional_fields).name = "queue capacity", (additional_fields).description = "max capacity of the queue"];
   
   // cache
-  CACHE_EVICTION = 10 [(additional_fields).name = "cache eviction", (additional_fields).description = "cache eviction count"];
```

Comment:
Was there a reason for changing these old values? We try not to change old protobuf values. I think `8-10` were intentionally left unused to add more queue metrics later.

Reply:
My bad. When I changed it back, got my numbers wrong. Fixed it.

Reply:
Fixed it.